crates/eips/Cargo.toml

```diff
@@ -70,6 +70,7 @@ serde = [
 kzg = ["kzg-sidecar", "sha2", "dep:derive_more", "dep:c-kzg", "dep:once_cell"]
 kzg-sidecar = ["sha2"]
 sha2 = ["dep:sha2"]
+k256 = ["alloy-primitives/k256"]
```

Comment:
Would be good to have `seck256k1` lib/feature flag as it is faster. But looking at it as a low priority.